express-nodejs/src/index.ts

```diff
@@ -24,14 +28,52 @@ function uploadFileToGCS(upFile: Express.Multer.File): string {
     });
     stream.on('finish', () => {
         console.log('<GCS>upload file');
+        asyncRecognizeGCS("gs://example_backet/" + fileName);
     });
     stream.end(upFile.buffer);
     return fileName;
 }
 
+function outputTextFile(text: string) {
+    fs.writeFileSync('test.txt', text);
+}
+
+
+async function asyncRecognizeGCS(gcsURI: string) {
```

Comment:
関数名の先頭に async つけるよりは後ろの方が良いです (C# の慣例だけど)
または何もつけない

また RecgnizeGCS だと「GCS を認識する」となり、意味が通りません。
```suggestion
async function speechToText(textFileUri: string) {
```

Reply:
関数名を変更します。
引数についてはgcs上のuriなのでこのように書いたのですが分かりづらいですか？
https://github.com/googleapis/nodejs-speech/blob/master/samples/recognize.js
公式だとこの変数名で書いているみたいです(ローカルファイルの場合と区別する為とは思いますが...)

Reply:
自分だけかもしれないけど、公式のサンプルコードも読みづらいですね。
おそらく、SpeechToText のサンプルコード (= 扱う対象が明確) であり、かつ、gcsUri を追っていくと audio = {uri: gcsUri} のように音声データであることがわかるようになっている、また、ローカルファイルを扱うケースもあることからそのようにしているのだとは思いますが...

Reply:
関数名変更しました　
別プルリクエストにまとめました。#9